README.md

```diff
@@ -1 +1,73 @@
-# videojs-verify-generator-plugin
+# videojs-generator-verify
+
+[![Build Status](https://travis-ci.org/brightcove/videojs-generator-verify.svg?branch=master)](https://travis-ci.org/brightcove/videojs-generator-verify)
+[![Greenkeeper badge](https://badges.greenkeeper.io/brightcove/videojs-generator-verify.svg)](https://greenkeeper.io/)
+[![Slack Status](http://slack.videojs.com/badge.svg)](http://slack.videojs.com)
+
+[![NPM](https://nodei.co/npm/videojs-generator-verify.png?downloads=true&downloadRank=true)](https://nodei.co/npm/videojs-generator-verify/)
+
+A tool to verify that a videojs-generate-plugin project is ready for publish.
```

Comment:
Should this be `generator-videojs-plugin`?

Reply:
yep